### PR TITLE
fix(ui): Limit issue title to one line for message

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

**Comment by dcramer** [comment]
This is specifically to make this kind of page a bit more bearable: https://sentry.io/sentry/sentry/issues/744193755/

File: src/sentry/static/sentry/less/group-detail.less [hunk 1]
```diff
@@ -44,6 +44,25 @@
     border-radius: 15px;
   }
 
+  .event-message {
+    position: relative;
+
+    .error-level {
```
Comment (dcramer) [inline src/sentry/static/sentry/less/group-detail.less]: this fixes another issue where a long message forces the error level to wrap on a new line as well

**Comment by dcramer** [comment]
screen shot 2018-10-26 at 12 40 45 pm

**Comment by dcramer** [comment]
I gotta run to an appointment but if someones feeling an itch they can commandeer/merge this after review/green.

**Review by ckj — APPROVED** [review]

File: src/sentry/static/sentry/less/group-detail.less (hunk 1 above)
Comment (ckj) [reply src/sentry/static/sentry/less/group-detail.less]: Looks like we hide the level-indicator on the issue share page... Digging in now to see if there's a quick fix. https://percy.io/getsentry/sentry/builds/1118989/view/69878739/1280?browser=firefox&mode=diff

**Pull request merged**